votes/initiateNewVote/_EDIT_ME.yml

```
@@ -24,8 +24,13 @@ headerInstructions: |
 #    voters express their preference for each candidates, no matter how many
 #    there are.
 candidates:
-  - TODO
-  - TODO
+  - Throw when passed to `--env-file` flag, provide no other flag (status quo).
+  - Ignore when passed to `--env-file` flag, provide no other flag.
+  - Warn when passed to `--env-file` flag, provide no other flag.
+  - Throw when passed to `--env-file` flag, warn when passed to a new `--env-file-if-exists`.
+  - Throw when passed to `--env-file` flag, do nothing when passed to a new `--env-file-if-exists`.
```

For the sake of completeness 
```suggestion
  - Throw when passed to `--env-file` flag, ignore when passed to a new `--env-file-if-exists`.
  - Ignore when passed to `--env-file` flag, throw when passed to a new `--env-file-must-exist`.
```